docs/getting-started/products/ylockers/yprisma/yprisma-faq.md

```diff
@@ -0,0 +1,83 @@
+# yPRISMA FAQ
+
+## Where can I buy yPRISMA?
+
+yPRISMA can be purchased through the following platforms:
+
+* If you already have PRISMA tokens, you can ZAP directly into yPRISMA using Yearn's yPRISMA zapper contract [**here**](add link)
+* [Curve Finance](https://curve.fi/#/ethereum/swap?from=0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee&to=0xe3668873d944e4a949da05fc8bde419eff543882) - The primary liquidity source for yPRISMA
+* [CowSwap](https://swap.cow.fi/#/1/swap/ETH/YPRISMA) - DEX aggregators like CowSwap should also work well
+
+## Should I stake my yPRISMA directly or auto-compound it in the vault?
+
+That's totally up to you and depends on your personal situation and preferences. Here are a few things to consider:
+
+* **Max Boost:**
+  * Direct stakers will need to wait four weeks before their yPRISMA is max boosted and earning the maximum staking APR displayed.
+  * Vault depositors join a single shared boost position, but this is unlikely to reach the maximum and will adjust up and down as other users deposit and withdraw.
+* **Rewards:**
+  * Direct stakers receive their yield in stablecoins making them less exposed to the price fluctuations of the underlying governance token and the yPRISMA peg.
+  * Vault depositors are "all in" on yPRISMA.
+* **Gas Fees:**
+  * Direct stakers must manually claim their mkUSD stablecoin rewards - which requires gas and can be expensive.
```

Comment:
I would add something here about how rewards for direct takers continue to accumulate even if users don't claim every week, though. And since they're vault tokens their yield is earning yield.

Reply:
updated the wording:
> Direct stakers must manually claim their mkUSD stablecoin rewards. Although earned rewards automatically accumulate and earn interest, this requires at least one additional transaction and can be expensive, depending on gas pricing and network congestion.